cardano-testnet/src/Test/Assert.hs

```diff
@@ -0,0 +1,52 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Assert
+  ( readJsonLines
+  , assertChainExtended
+  ) where
+
+import Control.Lens ((^.))
```

Comment:
Have a look at the other test modules (in cardano-cli/cardano-node) and follow how the modules are organized there.

Reply:
I had forgotten we use `stylish-haskell` since moving onto my M1 Macbook where it isn't set up.  😅

Hopefully this is all sorted now?